examples/news_alerts/kafka_utils.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/log;
+import ballerina/uuid;
+import ballerinax/kafka;
+
+configurable decimal POLL_INTERVAL = 100.0;
+
+type NewsProducerRecord record {|
+    *kafka:AnydataProducerRecord;
+    string key;
+    NewsRecord value;
+|};
+
+type NewsConsumerRecord record {|
+    *kafka:AnydataConsumerRecord;
+    NewsRecord value;
+|};
+
+final kafka:Producer producer = check new (kafka:DEFAULT_URL);
+
+isolated function produceNews(NewsUpdate newsUpdate) returns NewsRecord|error {
+    string id = uuid:createType1AsString();
+    Agency agency = check getAgency(newsUpdate.publisherId);
+    NewsRecord news = {id: id, ...newsUpdate};
+    NewsProducerRecord producerRecord = {
```

Comment:
Instead of defining a separate `NewsProducerRecord` we can also directly send the values to the `send` method.